js/Loader.js

```diff
@@ -81,6 +81,21 @@ Loader.prototype.handleLoad = function(doc, fileNameOrUrl, options) {
                 } 
                 console.log(obj.timestamp);
             } else if (desc.type === 'osmDiff') {
+                var remarks = doc.getElementsByTagName('remark')
+                if (remarks.length) {
```

Comment:
Just to be safe I suggest to change it to

`if (remarks && remarks.length) {`

Reply:
Good point. Thank you. It was too late yesterday evening ;).